presto-docs/README.md

```diff
@@ -45,6 +45,8 @@ Sphinx and the included `make` script.
 Sphinx installation instructions for various operating systems and packaging
 systems are [available on the Sphinx site](https://www.sphinx-doc.org/en/master/usage/installation.html).
 
+The `pip`-based installation is preferred by the Presto team.
```

Comment:
I think this does not belong to this commit. Also I don't think this is something that should be mentioned and I was not aware of such preference.

Reply:
I added it since sphinx-autobuild is best installed with pip .. I will move it down.

Reply:
done